src/i18n/pl/nav.ts

```diff
@@ -1,55 +1,86 @@
+	'guides/view-transitions': 'View Transitions',
+	'guides/prefetch': 'Prefetch',
+	'guides/dev-toolbar': 'Narzędzia deweloperskie',
+
+	'addons': 'Integracje',
+	'guides/integrations-guide': 'Dodaj integrację',
+	'guides/framework-components': 'Frameworki UI',
+	'guides/server-side-rendering': 'Renderowanie po stronie serwera (SSR)',
+	'guides/astro-db': 'Astro DB',
+
+	'examples': 'Przykłady',
+	'guides/migrate-to-astro': 'Migracja do Astro',
+	'recipes/studio': 'Połączenie z Astro Studio',
+	'guides/cms': 'Połączenie z CMS',
+	'guides/backend': 'Użycie usług backendowych',
 	'guides/deploy': 'Wdrażanie',
-	features: 'Przewodniki',
+	'guides/recipes': 'Więcej przykładów',
+
+	'features': 'Przewodniki',
+	'guides/routing': 'Routing',
+	'guides/markdown-content': 'Markdown',
+	'guides/client-side-scripts': 'Skrypty i obsługa zdarzeń',
 	'guides/styling': 'Stylowanie i CSS',
-	'guides/framework-components': 'Frameworki UI',
+	'guides/images': 'Obrazy',
+	'guides/fonts': 'Czcionki',
 	'guides/imports': 'Importowanie',
-	'guides/integrations-guide': 'Integracje',
-	'guides/server-side-rendering': 'Renderowanie po stronie serwera (SSR)',
+	'guides/endpoints': 'Endpointy',
 	'guides/data-fetching': 'Pobieranie danych',
-	'guides/publish-to-npm': 'Publikowanie na NPM',
-	// 'recipes/sharing-state-islands': 'Dzielenie stanu',
-	// 'guides/rss': 'RSS',
-	'guides/troubleshooting': 'Rozwiązywanie problemów',
-	'guides/cms': 'Łączenie z CMS',
-	'guides/images': 'Obrazy',
-	// 'guides/fonts': 'Czcionki',
+	
+	'guides/internationalization': 'Internacjonalizacja',
+	'guides/middleware': 'Middleware',
+	'guides/ecommerce': 'E-commerce',
 	'guides/testing': 'Testowanie',
-	'guides/client-side-scripts': 'Skrypty i zdarzenia',
-	configuration: 'Konfiguracja',
+	'guides/authentication': 'Autentykacja',
+	'guides/troubleshooting': 'Rozwiązywanie błędów',
+		
+	'configuration': 'Konfiguracja',
 	'guides/configuring-astro': 'Plik konfiguracyjny Astro',
 	'guides/typescript': 'TypeScript',
 	'guides/aliases': 'Aliasy importowania',
 	'guides/environment-variables': 'Zmienne środowiskowe',
-	reference: 'Referencja',
-	'reference/directives-reference': 'Dyrektywy szablonów',
-	'reference/adapter-reference': 'API adapterów',
-	'reference/integrations-reference': 'API integracji',
-	'reference/cli-reference': 'Interfejs linii poleceń',
-	'reference/api-reference': 'API runtime',
+
+	'reference': 'Odnośniki',
 	'reference/configuration-reference': 'Konfiguracja',
-	'reference/error-reference': 'Referencja błędów',
+	'reference/api-reference': 'API środowiska wykonawczego',
+	'reference/integrations-reference': 'API integracji',
+	'reference/adapter-reference': 'API adapterów',
+	'reference/image-service-reference': 'API usług obrazów',
+	'reference/dev-toolbar-app-reference': 'API narzędzi deweloperskich',
+	'reference/directives-reference': 'Dyrektywy szablonów',
+	'reference/cli-reference': 'Interfejs linii poleceń Astro',
```

Comment:
Technically it will be a repetition (CLI -> command-line interface -> interfejs command-line interface). That's why I didn't use CLI abbreviation. On the other hand, such constructs are commonly used in spoken language (interfejs API). 

Hard to say which option is better 🤔

Reply:
Yeah, you do have a point.

> On the other hand, such constructs are commonly used in spoken language (interfejs API).

That's what I was thinking.

Will ask in Discord then. Let's see what others think

Reply:
We got no answers (🥲) so I'd leave it as is and move on